### PR TITLE
checked out a branch with the correct naming conventions and push it up with our recent changes

### PR DESCRIPTION
**Pull request by omarbowen01**

### CODE REVIEW

**Review by WalkerTexas-coder — CHANGES_REQUESTED** [review]
You are missing an assets directory for the PR. 
Add the missing directory and I will approve.

**Comment by omarbowen01** [comment]
Assets directory is there under src, not sure why it didn't reflect in the changes we pushed up.

File: src/pages/NotFound.js [hunk 1]
```diff
@@ -3,7 +3,11 @@ import React, { Component } from 'react'
 class NotFound extends Component {
   render () {
     return (
-      <div>Footer</div>
+      <div>
+      <div class="img3">
+        <img src="https://c8.alamy.com/zooms/9/d4f9b47ecab74f9aa14d27950751d5ca/tr609x.jpg" height="auto" width="auto"  />
+      </div>
```
Comment (WalkerTexas-coder) [inline src/pages/NotFound.js:9]: fix up indentation and I will approve

**Review by WalkerTexas-coder — APPROVED** [review]
Lets GOOOOOOOOOO!

**Pull request merged**